recipes/systemc/meta.yaml

```
@@ -0,0 +1,117 @@
+          - {{ compiler('cxx') }}                                # [not win]
+          - {{ pin_subpackage('systemc-doc', exact=True) }}      # [not win]
+          - make                                                 # [not win]
+        commands:                                                # [not win]
+          - test "$SYSTEMC_HOME" == "$PREFIX"                    # [not win]
+          - cp -r "$PREFIX"/share/doc/systemc/examples .         # [not win]
+          # patch Makefile.rules because we want to use the installed libsystemc  
+          # AND we want to use the dynamic lib which the linker should be able to find
+          - "sed -i 's|$(EXE): $(OBJS) $(SYSTEMC_LIB_DIR)/libsystemc.a|$(EXE): $(OBJS)|' examples/build-unix/Makefile.rules"  # [not win]
+          - make -C examples/build-unix -j $CPU_COUNT run        # [not win]
+          - make -C examples/build-unix -j $CPU_COUNT check      # [not win]
+    - name: systemc                                              # [not win]
+      requirements:                                              # [not win]
+          run:                                                   # [not win]
+              - {{ pin_subpackage('systemc-dev', exact=True) }}  # [not win]
+              - {{ pin_subpackage('systemc-doc', exact=True) }}  # [not win]
+
+# TODO Add tests for windows
+#test:                         # [win]
+#  requires:                   # [win]
+#    - {{ compiler('cxx') }}   # [win]
+#  commands:                   # [win]
+#    - cd examples/build-msvc  # [win]
+#    - nmake /nologo CONFIG=Release PLATFORM=x64 VCBUILD=vcbuild check  # [win64]
+#    - nmake /nologo CONFIG=Release PLATFORM=Win32 VCBUILD=vcbuild check  # [win32]
+
+about:
+  home: https://systemc.org/
+  summary: 'The language for System-level design, modeling and verification'
+  description: |
+    SystemC(TM) addresses the need for a system design and verification language that spans hardware
+    and software. It is a language built in standard C++ by extending the language with the use of
+    class libraries. The language is particularly suited to model system's partitioning, to evaluate
+    and verify the assignment of blocks to either hardware or software implementations, and to
+    architect and measure the interactions between and among functional blocks. Leading companies in
+    the intellectual property (IP), electronic design automation (EDA), semiconductor, electronic
+    systems, and embedded software industries currently use SystemC for architectural exploration,
+    to deliver high-performance hardware blocks at various levels of abstraction and to develop
+    virtual platforms for hardware/software co-design. SystemC has been standardized by the Open
+    SystemC Initiative (OSCI) and Accellera Systems Initiative and ratified as IEEE Std. 1666(TM)-2011.
+  license: Apache-2.0
+  license_family: Apache
+  license_file:
+    - LICENSE
+    - NOTICE
```

I think https://github.com/accellera-official/systemc/blob/39740075f47fedbce242808d357fcfb6d3d33957/src/sysc/packages/boost/LICENSE_1_0.txt#L13-L15 means that we don't _have_ to include the boost license since they are only using some of the header-only libraries.  But, I'm not a lawyer and I'm going to err on the side of including more than less:
```suggestion
    - NOTICE
    - src/sysc/packages/boost/LICENSE_1_0.txt
```